report/21_Process-Developer-Interaction.md

```diff
@@ -1 +1,5 @@
-How do you interact as developers?
+The developers interact mainly through a weekly meeting right after the lecture, through the use of a Discord Server, and through the use of the GitHub Repository on which the project is hosted. After the lockdown caused be Covid-19, the weekly meeting started being held over Discord as well. 
```

Comment:
Can we change: 

> The developers interact mainly through a weekly meeting right after the lecture, through the use of a Discord Server, and through the use of the GitHub Repository on which the project is hosted. After the lockdown caused be Covid-19, the weekly meeting started being held over Discord as well. 
to :

The developers interacted mainly through a weekly meeting right after the lecture where tasks were decided and issues on Github made for each of them. Other communication was through the use of a Discord Server, and issues on the GitHub Repository where the project is hosted. After the lockdown caused be Covid-19, the weekly meeting started being held over Discord as well, but no other major changes were made. 

the "and issues on Github made for each of them." could possibly be left out.